### PR TITLE
dev-util/kubebuilder: new package (2.0.2)

### PR DESCRIPTION
**Pull request by dmolik**
Kubebuilder is a CLI Tool to generate scaffolding for kubernetes
operators.

Package-Manager: Portage-2.3.77, Repoman-2.3.17
Signed-off-by: Dan Molik <dan@danmolik.com>

### CODE REVIEW

File: dev-util/kubebuilder/kubebuilder-2.0.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	"gopkg.in/yaml.v2 v2.2.2 github.com/go-yaml/yaml"
+)
+
+SRC_URI="https://github.com/kubernetes-sigs/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	$(go-module_vendor_uris)"
+
+LICENSE="Apache-2.0 BSD-4 ECL-2.0 JSON MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
```
Comment (ConiKost) [inline dev-util/kubebuilder/kubebuilder-2.0.1.ebuild]: Drop it, if empty.

File: dev-util/kubebuilder/kubebuilder-2.0.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+DEPEND=""
```
Comment (ConiKost) [inline dev-util/kubebuilder/kubebuilder-2.0.1.ebuild]: Drop it, if empty.

File: dev-util/kubebuilder/kubebuilder-2.0.1.ebuild (hunk 1 above)
Comment (dmolik) [reply dev-util/kubebuilder/kubebuilder-2.0.1.ebuild]: done

File: dev-util/kubebuilder/kubebuilder-2.0.1.ebuild (hunk 2 above)
Comment (dmolik) [reply dev-util/kubebuilder/kubebuilder-2.0.1.ebuild]: done

**Comment by mrueg** [comment]
Thanks, applied as 3edfd542777749cc9554dc1db0072f524aff7724

**Pull request closed without merging**